.github/CODEOWNERS

```
@@ -1,4 +1,4 @@
 # The renovate-approve bot is allowed to merge its own PRs.  As it's not
 # a regular GitHub user (but rather, an app!), its username is indicated
 # as such, using the `app/` prefix. https://github.com/apps/renovate-approve
-/docs/ app/renovate-approve @stephenbarlow @glasser
+/docs/ app/renovate-approve @apollographql/docs @glasser
```

Ninja putting myself in there as well
```suggestion
/docs/ app/renovate-approve @apollographql/docs @glasser @trevor-scheer
```